main.js

```diff
@@ -52,4 +52,64 @@ for (let i = 0; i < arr10.length; i++) {
     if (arr10[i] % 2 === 0) {
         console.log(arr10[i]);
     }
+}
+
+//Lesson 16
+//task 16.5
+let str = 'шалаш';
+let str2 = 'Ирина';
+const  polyndrome = function (poly) {
+    let res = false;
+    for (let i = 0; i <= poly.length; i++) {
+        if (poly[i] !== poly[poly.length - (i + 1)]) {
+            res = false;
+            break;
+        } else
+            res = true;
+    }
+    return res;
+};
+
+console.log(polyndrome(str));
+console.log(polyndrome(str2));
+
+//task 16.6
+let num5 = 5;
+let num10 = 10;
```

Comment:
The worse naming of variables.

Reply:
I followed the example in the lecture.

Reply:
Great if you will not be confused with a different value of num5 somewhere in the depth of the code. But if you will write code not for yourself only, it is good to avoid such names.